tornjak-frontend/src/components/agent-list.js

```diff
@@ -97,11 +106,12 @@ class AgentList extends Component {
       <div>
         <h3>Agents List</h3>
         {this.props.globalErrorMessage !== "OK" &&
-          <div className="alert-primary" role="alert">
-            <pre>
-              {this.props.globalErrorMessage}
-            </pre>
-          </div>
+          <InlineNotification kind="error" title="Error" subtitle={this.props.globalErrorMessage} />
```

Comment:
probably once this gets fleshed out to abstract into its own error class since it will need to be repeated elsewhere.